features/largeVideo/components/LargeVideo.js

```diff
@@ -19,10 +22,23 @@ class LargeVideo extends Component {
      * @returns {ReactElement}
      */
     render() {
+        const avatar = this.props.avatar;
+        const videoTrack = this.props.videoTrack;
+        const renderAvatar
+            = typeof avatar !== 'undefined'
+                && avatar !== ''
+                && (!videoTrack
+                    || (!videoTrack.videoStarted || videoTrack.muted));
```

Comment:
You removed the unnecessary check of videoTrack. Thank you!

Now though it's quite obvious that the parenthesis are unnecessary. I'd day that they give a false impression that the boolean expression is more complex than it actually is because I see the opening bracket and I immediately switch to parsing another level of a boolean expression only to discover at the closing bracket that there's no other level. (I'll be taking care of this myself again.)